### ISSUE TITLE
Bad Automatic Selection with Enum for Write

### ISSUE THREAD

**Issue by mxrmesulog** [LabVIEW-DCAF/tagbus#20]
Hello,

I'm Using the DCAF Framework, and when scripting my module, I discover a bug with enum.

the Automatic polimorphic Instance is choosing I8, I16 or I32 for Enum thatn can be only U8, U16 or U32.
this can be fix by changing the order in the polymorphic Instance and placing Unsigned Number before signed Number.

I'm Using labVIEW 2017 SP1 f3 in english 32 bits. OS Windows 10 + NI Tag Bus v1.4.0.3 installed from VIPM.

the Initial Version 
![initial version](https://user-images.githubusercontent.com/45432999/49168458-d198c100-f337-11e8-9e8e-1f798bba460c.png)

Fixed Version
![fixed version](https://user-images.githubusercontent.com/45432999/49168555-073daa00-f338-11e8-844c-ae49b9a5604e.png)

Best regards

**Comment by becega** [comment]
You are correct, this bug is easy to reproduce, it might be more a LabVIEW bug than a DCAF, but I will look into adding the patch for DCAF just need to be sure making the change has no other effects. Thanks for submitting it.

**Comment by mxrmesulog** [comment]
Hier. Yes this id a labview bug. I bref tout truc it on 2013 ans report it. Best regards
